### PR TITLE
add patch license

### PR DESCRIPTION
**Pull request by ck-developer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/ArrowSphere/nodejs-api-client.git"
   },
-  "version": "3.82.0",
+  "version": "3.83.0-rc.ckh.1",
```
Comment (fdianoux) [inline package.json:7]:
```suggestion
  "version": "3.83.0",
```

File: src/licenses/licensesClient.ts [hunk 2]
```diff
@@ -577,6 +584,23 @@ export class LicensesClient extends AbstractRestfulClient {
...
+    parameters: Parameters = {},
+  ): Promise<void | PartialResponse> {
+    this.path = `/${licenseReference}`;
+
+    const response = await this.patch<undefined | PartialResponseData>(
+      payload,
+      parameters,
+    );
+
+    if (response && response.status == 202) {
```
Comment (fdianoux) [inline src/licenses/licensesClient.ts:599]:
```suggestion
    if (response && response.status == constants.HTTP_STATUS_PARTIAL_CONTENT) {
```

File: tests/licenses/licensesClient.test.ts [hunk 3]
```diff
@@ -741,6 +759,41 @@ describe('LicensesClient', () => {
...
+      );
+
+      expect(response).to.not.instanceOf(PartialResponse);
+      expect(nock.isDone()).to.be.true;
+    });
+
+    it('calls the get method and returns its result with partial success', async () => {
+      nock(LICENSES_MOCK_URL)
+        .patch(LICENSE_MOCK_URL_LICENSE)
+        .reply(202, PARTIAL_RESPONSE_UPDATE_LICENSE);
```
Comment (fdianoux) [inline tests/licenses/licensesClient.test.ts:782]:
```suggestion
        .reply(constants.HTTP_STATUS_PARTIAL_CONTENT, PARTIAL_RESPONSE_UPDATE_LICENSE);
```

File: tests/licenses/licensesClient.test.ts [hunk 4]
```diff
@@ -741,6 +759,41 @@ describe('LicensesClient', () => {
     });
   });
 
+  describe('updateLicense', () => {
+    const getLicenseClient = new PublicApiClient()
+      .getLicensesClient()
+      .setUrl(LICENSES_MOCK_URL);
+
+    it('calls the get method and returns its result', async () => {
+      nock(LICENSES_MOCK_URL).patch(LICENSE_MOCK_URL_LICENSE).reply(204);
```
Comment (fdianoux) [inline tests/licenses/licensesClient.test.ts:768]:
```suggestion
      nock(LICENSES_MOCK_URL).patch(LICENSE_MOCK_URL_LICENSE).reply(constants.HTTP_STATUS_NO_CONTENT);
```

File: tests/licenses/licensesClient.test.ts [hunk 5]
```diff
@@ -302,6 +312,14 @@ const PAYLOAD_UPDATE_SEATS_JOI = Joi.object({
   [LicenseGetFields.COLUMN_SEATS]: Joi.number(),
 });
 
+const PARTIAL_RESPONSE_UPDATE_LICENSE = {
+  [PartialResponseFields.COLUMN_STATUS]: 202,
```
Comment (fdianoux) [inline tests/licenses/licensesClient.test.ts:316]:
```suggestion
  [PartialResponseFields.COLUMN_STATUS]: constants.HTTP_STATUS_PARTIAL_CONTENT,
```

**Comment by fdianoux** [comment]
Need to add a comment in CHANGELOG.MD

**Review by fdianoux — APPROVED** [review]

**Review by lth-arrow — APPROVED** [review]

**Pull request merged**